README.md

```diff
@@ -75,7 +106,10 @@ the variable pizza inside your method.
 
 Your Answer:
 ```js
-// write code here
+var pizza =
+temperature: "70";
+bake: "";
```

Comment:
this should include a method that bakes the pizza to 300 degrees. so something like:

```
bake: function(){
    this.temperature = 300;
  }
```